### PR TITLE
Update distro_signatures.json

### PR DESCRIPTION
**Pull request by nthrow**
Now featuring Xenial Xerxes

### CODE REVIEW

File: config/cobbler/distro_signatures.json [hunk 1]
```diff
@@ -445,6 +445,22 @@
     "kernel_options_post":"",
     "boot_files":[],
     "boot_loaders":{"ppc64el":["grub2"]}
+   },
+   "xenial": {
+    "signatures":["dists", ".disk"],
+    "version_file":"Release|mini-info",
+    "version_file_regex":"Archive: xenial",
```
Comment (guessi) [inline config/cobbler/distro_signatures.json]: can't find "Archive: xenial" in `Release`, or `mini-info`.
should be: `"version_file_regex":"Codename: xenial|Ubuntu 16.04",`

File: config/cobbler/distro_signatures.json [hunk 2]
```diff
@@ -445,6 +445,22 @@
...
+    "kernel_arch_regex":null,
+    "supported_arches":["i386","amd64"],
+    "supported_repo_breeds":["apt"],
+    "kernel_file":"linux(.*)",
+    "initrd_file":"initrd(.*)\\.gz",
+    "isolinux_ok":false,
+    "default_kickstart":"/var/lib/cobbler/kickstarts/sample.seed",
+    "kernel_options":"",
+    "kernel_options_post":"",
+    "boot_files":[]
```
Comment (guessi) [inline config/cobbler/distro_signatures.json]: missing `boot_loaders` here

**Comment by jmaas** [comment]
Support added to master, release28 and release26 branches

**Comment by jmaas** [comment]
Also updated the online signature repository, so you should be able to get support with "cobbler signature update"

**Pull request closed without merging**